src/Microsoft.ML.ModelBuilder.SearchSpace/Attribute/RangeAttribute.cs

```diff
@@ -0,0 +1,47 @@
+﻿// <copyright file="RangeAttribute.cs" company="Microsoft">
+// Copyright (c) Microsoft. All rights reserved.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.ML.ModelBuilder.SearchSpace.Option;
+
+namespace Microsoft.ML.ModelBuilder.SearchSpace
+{
+    [AttributeUsage(AttributeTargets.Property | AttributeTargets.Field, Inherited = false, AllowMultiple = false)]
+    public class RangeAttribute : Attribute
```

Comment:
Currently this supports `int`, `single`, `double`. Do we need to support the other int types?

Reply:
For now it seems that `int` is enough but we can add other types later if it's necessary

Reply:
### Resolved